src/Model/Metadata.php

```diff
@@ -55,6 +55,7 @@ public function offsetExists($offset): bool
     /**
      * @inheritDoc
      */
+    #[\ReturnTypeWillChange]
```

Comment:
Both methods have a return type, what issue is this fixing? Maybe we should fix the return type instead.
The commit message and the PR title should be a bit more descriptive.

Reply:
It was fix for the following warnings:
```
PHP Deprecated:  Return type of OpenEuropa\EuropaSearchClient\Model\Metadata::offsetGet($offset) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /builds/digit/ewcms/vendor/openeuropa/europa-search-client/src/Model/Metadata.php on line 58
PHP Deprecated:  Return type of OpenEuropa\EuropaSearchClient\Model\Metadata::offsetSet($offset, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /builds/digit/ewcms/vendor/openeuropa/europa-search-client/src/Model/Metadata.php on line 67
```

Reply:
I see but in the warning this `Return type of OpenEuropa\EuropaSearchClient\Model\Metadata::offsetGet($offset) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed` makes me think the build where it was triggered was not using the proper version of `europa-search-client` because it seems like there is no return type and in fact the method has a return type and it is compatible with ArrayAccess::offsetGet(mixed $offset): mixed.